tools/build.mjs

```diff
@@ -1,10 +1,28 @@
+import esbuildCopyPlugin from "@sprout2000/esbuild-copy-plugin";
+
 import concurrently from "concurrently";
 import esbuild from "esbuild";
 
 console.log("start");
 const dev = process.argv[2];
 
 const plugins = [
```

Comment:
this plugin is only needed for `nodeBuildOptions`, not `browserBuildOptions`. The 2 run concurrently (see line 88 below) thus will produce errors.
Actually you can simply do `fs.copyFile` with native Nodejs after build done (for example at line 100 below)